common/lib/xmodule/xmodule/graders.py

```diff
@@ -341,7 +344,7 @@ def total_with_drops(breakdown, drop_count):
                     possible=float(possible)
```

Comment:
Nit: Instead of coercing `possible` to a float, above use `from __future__ import division`.  Then you get floating point division with `/` and a rounded-down integer result with `//`, regardless of your input values.  This is also the behavior in python 3, so using it now helps future-proof the code.